onnxscript/tests/function_libs/torch_aten/ops_correctness_test.py

```diff
@@ -909,6 +916,17 @@ def _where_input_wrangler(
         matcher=lambda sample: len(sample.kwargs) == 0 or sample.kwargs.get("p", 0.0) > 0.0,
         reason="dropout is random so the result not match",
     ),
+    skip(
+        "nn.functional.grid_sample",
+        # FIXME: check why ort's GridSample() return different result than Torch
+        matcher=lambda sample: sample.kwargs.get("mode") == "bicubic",
+        reason="'bicubic' mode in ORT implemented differently with Torch",
```

Comment:
```suggestion
        reason="fixme: 'bicubic' mode in ORT implemented differently with Torch",
```

Reply:
Torch implemented this using the cubic convolution algorithm with alhpa=−0.75, might be different than ORT